README.md

```diff
@@ -80,22 +80,7 @@ repo](https://github.com/theupdateframework/specification/issues).
 Security Issues and Bugs
 ------------------------
 
-Security issues can be reported by emailing jcappos@nyu.edu.
-
-At a minimum, the report must contain the following:
-
-* Description of the vulnerability.
-* Steps to reproduce the issue.
-
-Optionally, reports that are emailed can be encrypted with PGP.  You should use
-PGP key fingerprint **E9C0 59EC 0D32 64FA B35F  94AD 465B F9F6 F8EB 475A**.
-
-Please do not use the GitHub issue tracker to submit vulnerability reports.
-The issue tracker is intended for bug reports and to make feature requests.
-Major feature requests, such as design changes to the specification, should
-be proposed via a
-[TUF Augmentation Proposal](https://theupdateframework.github.io/specification/latest/#tuf-augmentation-proposal-tap-support)
-(TAP).
+See [SECURITY.md](SECURITY.md)
```

Comment:
Should we move `SECURITY.md` to `docs/`? All our `*.md` files are already there and[ GitHub will still find it](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository#about-security-policies):
  > To give people instructions for reporting security vulnerabilities in your project, you can add a SECURITY.md file to your repository's root, **docs**, or .github folder.

We'd have to adopt the change here...
```suggestion
See [SECURITY.md](docs/SECURITY.md)
```


 Related issue: #1849